hangman.css

```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css?family=Italianno&display=swap');
```

Comment:
Overall, I would like you to keep working on this project. I see your still developing your HTML, Javascript and CSS skills. Keep up writing down your plans before you code.  If you would like any help, hit me up on Slack so we can schedule some time.